.github/dependabot.yml

```diff
@@ -3,7 +3,12 @@ updates:
   - package-ecosystem: 'github-actions'
     directory: '/'
     schedule:
-      interval: 'weekly'
+      interval: 'monthly'
+    groups:
+      github-actions:
+        patterns:
+          - '*'
```

Comment:
i wouldn't do that - it makes it super confusing since you are combining so many different things together. if you really want to - then group "actions/*" (since those are all from GitHub) and leave everything else ungroup (since they are all unrelated to each other)